classes/class-wc-connect-shipping-label.php

```diff
@@ -106,36 +115,61 @@ protected function get_selected_rates( WC_Order $order ) {
 			$packages = json_decode( $shipping_method[ 'wc_connect_packages' ], true );
 			$rates = array();
 
-			foreach( $packages as $package ) {
+			foreach( $packages as $idx => $package ) {
 				if ( ! $package[ 'service_id' ] ) {
 					return array();
 				}
-				$rates[] = $package[ 'service_id' ];
+				$id = isset( $package[ 'id' ] ) ? $package[ 'id' ] : "box_{$idx}";
+				$rates[ $id ] = $package[ 'service_id' ];
 			}
 
 			return $rates;
 		}
 
-		protected function get_form_data( WC_Order $order ) {
-			$form_data = array();
+		protected function format_address_for_server( $address ) {
```

Comment:
"server" could be ambiguous to someone unfamiliar with the code. Would you consider "format_address_for_client_api" or something like that instead?